core/includes/install.core.inc

```diff
@@ -543,13 +547,21 @@ function install_tasks($install_state) {
   }
   $display_select_profile = $hidden_profile_count > 1 && count($install_state['profiles']) != 1;
 
+  // Determine whether a translation file must be downloaded during the
+  // 'install_download_translation' task. Download when a non-English language
+  // is selected, but no translation is yet in the translations directory.
+  $needs_download = isset($install_state['parameters']['langcode']) && !isset($install_state['translations'][$install_state['parameters']['langcode']]) && $install_state['parameters']['langcode'] != 'en';
```

Comment:
This line seems to fit the case of "Control structure conditions should also NOT attempt to win the Most Compact Condition In Least Lines Of Code Award™" (see [our coding standards](https://docs.backdropcms.org/php-standards#line-length)) 😅 ...since there's 3 instances of `$install_state['parameters']['langcode']` in it, can we please split that to a separate variable and then use that instead. It should make things easier to read/understand here.

Reply:
Fixed